test/extended/apibinding/apibinding.go

```diff
@@ -0,0 +1,46 @@
+package apibinding
+
+import (
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[area-apiexports]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		k = exutil.NewCLIWithWorkSpace("kcp-apibinding")
+	)
+
+	g.It("Author:pewang-Critical-[Smoke][API] Verify APIBinding working with personal workspace", func() {
+		// Shared compute could be only accessed from dev-provided test environments
+		// Skip for non-supported test environments
+		exutil.PreCheckEnvSupport(k, "kcp-stable.apps.kcp-internal", "kcp-unstable.apps.kcp-internal")
+		myWs := k.WorkSpace()
+
+		g.By("# Create an apibinding attach to the orgnaization workspace for the shared compute provided by acm should be failed")
+		myAPIBinding := NewAPIBinding(SetAPIBindingReferencePath("root:redhat-acm-compute"), SetAPIBindingReferenceExportName("kubernetes"))
+		myAPIBinding.CreateAsExpectedResult(k.WithOrgWorkSpaceServer(), false, `cannot get resource "apibindings" in API group "apis.kcp.dev" at the cluster scope`)
+
+		g.By("# Create an apibinding attach to a personal workspace for the shared compute provided by acm should be successful")
+		myAPIBinding.Create(k.WithSpecificWorkSpaceServer(myWs))
```

Comment:
@Phaow you could replace this with "Validate apibinding creation is successful in user home workspace"

Reply:
Sure. Addressed.